cli/testdata/coder_server_--help.golden

```diff
@@ -90,6 +90,9 @@ Use a YAML configuration file when your server launch become unwieldy.
       --prometheus-address host:port, $CODER_PROMETHEUS_ADDRESS (default: 127.0.0.1:2112)
           The bind address to serve prometheus metrics.
 
+      --prometheus-collect-agent-stats bool, $CODER_PROMETHEUS_COLLECT_AGENT_STATS
```

Comment:
I would say the opposite. It should be disabled by default so that when admin enables Prometheus metrics, they won't be extra charged. BTW I don't have a strong preference here, I'm ok with adjusting this behavior.

Reply:
I’m alright with leaving it disabled by default. I don’t really have a concept for how much extra data/load this introduces so if it makes sense to keep disabled by default let’s not change it.